mods/ra/weapons/other.yaml

```diff
@@ -185,9 +185,12 @@ TTankZap:
 
 DogJaw:
 	ValidTargets: Infantry
-	ReloadDelay: 10
-	Range: 3c0
+	ReloadDelay: 100
+	FireDelay: 200
 	Report: dogg5p.aud
+	Range: 1c512
+	Projectile: Bullet
+		Speed: 1c682
 	Warhead@1Dam: SpreadDamage
```

Comment:
We now do have it. https://github.com/OpenRA/OpenRA/pull/11715/commits/979b328e6e931c22d06480ea3e73b887cd6ef231